src/Modules/User/Components/Profile/UserDropdown/UserDropdown.tsx

```diff
@@ -177,6 +184,19 @@ class UserDropdownComponent extends BaseComponent<IProps, IState> {
       </Fragment>
     );
   }
+
+  private reloadPage() {
+    this.checkSignOutAtProfilePage();
+    window.location.reload();
+  }
+
+  private checkSignOutAtProfilePage() {
+    const userId = this.props.userInfo.id;
```

Comment:
`const { userInfo: { id } } = this.props;`

Reply:
I prefer using userId to id. It makes more sense.